socorro/scripts/es.py

```diff
@@ -68,6 +69,18 @@ def cmd_list():
         print("No indices.")
 
 
+def cmd_list_crashids(index):
+    """List crashids for index."""
+    es_conn = get_conn()
+    with es_conn() as conn:
+        search = Search(using=conn, index=index, doc_type=es_conn.get_doctype(),)
+        search = search.fields("processed_crash.uuid")
+        results = search.execute()
+        print("Crashids in %s:" % index)
+        for hit in results:
+            print(hit["processed_crash.uuid"][0])
```

Comment:
This is helpful since it prints all the crash ids in an ES index. It makes it easier to answer questions like, "The thing I'm looking for isn't coming up in search in this test. What's in there?"